src/controllers/repetitionController.js

```diff
@@ -18,9 +18,9 @@ const list = async (req) => {
   }
 };
 
-const remove = async (req) => {
+const deleteRepetition = async (req) => {
```

Comment:
Comme c'est dans le controlleur repetition, pas sure qu'il y ait besoin de l'appeler deleteRepetition (comme tu as fait dans le helper)

Reply:
idem qu'en front, on ne peut pas definir le nom de méthode `delete`, j'ai l'erreur suivante: `'delete' is not allowed as a variable declaration name.`